### PR TITLE
nits

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: static/img/shared/white-paper-dark.svg [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jjwill) [inline static/img/shared/white-paper-dark.svg:1]: I don't see anything referencing this file in this PR.

Comment (oriooctopus) [reply static/img/shared/white-paper-dark.svg:1]: It's used by libra-developer-plugins. Some image have to go in the main repo folder (libra-website, lips), so they go in the img/shared folder

Comment (jjwill) [reply static/img/shared/white-paper-dark.svg:1]: Weird, no way around that?

Comment (oriooctopus) [reply static/img/shared/white-paper-dark.svg:1]: I'll look into it

**Pull request merged**